### PR TITLE
Section 6.2 Tari transactions a simple explanation edited da

### PR DESCRIPTION
**Pull request by anselld**

### CODE REVIEW

**Comment by anselld** [comment]
Hi @CjS77, if you get a chance, please could you review my formatting and editing changes to your report.
I also have the following questions, please:
1. The heading of this section differs from the heading in the index in the online version. Which is correct?
2. Is it possible to change the 'black on blue' text to 'white on blue', to improve legibility for the reader?
3. The text in the header row of the first table (under 'Basic Transaction) is not centered (it is in the other tables). Is it possible to correct this?
4. The tables just above the 'Transaction Verification and Propagation' heading would look better if they were the same size. Is it possible to do this?
Please let me have you comments.
Thanks and regards
Denise

File: src/protocols/mimblewimble-1/MainReport.md [hunk 1]
```diff
@@ -227,58 +241,60 @@ and the combined aggregate signature, $$ s = s_a + s_b, R = R_a + R_b $$.
...
 
-#### All outputs have a valid range proof
-The range proof is verified against its matching commitment.
+   In this check, the node wants to make sure that no coins are created or destroyed in the transaction. But how can it 
+   do this if the values are blinded?
 
-### The values balance
-In this check, the node wants to make sure that no coins are created or destroyed in the transaction. But how can it do
-this if the values are blinded?
+   Recall that in an honest transaction, all the values (which are multiplied by _H_) cancel, and you're left with the
```
Comment (SWvheerden) [inline src/protocols/mimblewimble-1/MainReport.md]: They dont technically cancel, they equal the fees. The transaction as a whole:
Inputs on H = (Outputs on H) + (fee on H)

**Comment by CjS77** [comment]
> Hi @CjS77, if you get a chance, please could you review my formatting and editing changes to your report.
> I also have the following questions, please:
> 
> 1. The heading of this section differs from the heading in the index in the online version. Which is correct?
The one in the main text.

> 2. Is it possible to change the 'black on blue' text to 'white on blue', to improve legibility for the reader?
Only globally, and not from this report. Best is to create a new issue so that a designer / front-end dev can pick it up and apply.

> 3. The text in the header row of the first table (under 'Basic Transaction) is not centered (it is in the other tables). Is it possible to correct this?
Hmm, I'm not sure -- the formatting seems to indicate that all tables should be left-justifiied

> 4. The tables just above the 'Transaction Verification and Propagation' heading would look better if they were the same size. Is it possible to do this?
Afraid not. You typically only specify content and intent (e.g. empahsis) in Markdown; you don't control formatting.

File: src/protocols/mimblewimble-1/MainReport.md (hunk 1 above)
Comment (CjS77) [reply src/protocols/mimblewimble-1/MainReport.md]: That fact is mentioned above.

**Comment by anselld** [comment]
Hi @hansieodendaal, please could you let me know what the next step in the process is. Thanks.

**Pull request merged**